squad-server/plugins/discord-server-status.js

```diff
@@ -123,8 +123,15 @@ export default class DiscordServerStatus extends DiscordBaseMessageUpdater {
   async updateStatus() {
     if (!this.options.setBotStatus) return;
 
+    let players = this.server.a2sPlayerCount;
+    if (this.server.publicQueue || this.server.reserveQueue)
+      players += `+${this.server.publicQueue + this.server.reserveQueue}`;
+
+    let slots = this.server.publicSlots;
+    if (this.server.reserveSlots) slots += `+${this.server.reserveSlots}`;
+
     await this.options.discordClient.user.setActivity(
-      `(${this.server.a2sPlayerCount}/${this.server.publicSlots}) ${
+      `(${players}/${slots}) ${
```

Comment:
Recommended that this follow the rest of the repo. Ex making it `this.server.players` etc. To keep it consistent among the rest of repo.

Reply:
I agree with the pull request as is - it is consistent with the remainder of the plugin at least and I believe the values used are slightly more reliable that `this.servers.players` at times, e.g. during map changes.